src/node/node.service.ts

```diff
@@ -166,6 +166,38 @@ export class NodeService {
     return response.data.id;
   }
 
+  async signSponsorTransaction(sender: string, recipient: string): Promise<any> {
```

Comment:
This will only create a signed sponsor transaction, but the account won't be sponsored because the tx isn't broadcasted.

* Rename the method to `sponsorAccount`
* The sender is aways (the first wallet account of) the node. So you shouldn't need to specify it. Optionally you might add it to the config, so you can choose one of the wallet addresses.
* The signed tx (`response.data`) must be broadcasted

Reply:
Done